### DISCUSSION TITLE
Icon for Monitorr & Organizr

### DISCUSSION THREAD

**Discussion by SavageAUS** [sct/overseerr#432] [General]
![overseer](https://user-images.githubusercontent.com/17326832/102777962-96b3cb00-43e1-11eb-94f1-02fba2138a8d.png)
![overseer](https://user-images.githubusercontent.com/17326832/102777972-9ca9ac00-43e1-11eb-8da6-fe0d9c04f201.png)

No bugs to report just wanted to add image that looks great in Monitorr, yet to test in Organizr but should be fine.

**Comment by sct** [comment]
We also have our own icon! 

https://github.com/sct/overseerr/blob/develop/public/os_logo_square.png

**Reply by SavageAUS** [reply]
My bad I didn’t see it.
